docs/getting-started/slack.md

```diff
@@ -1,6 +1,7 @@
-With everything in place to get started with Atomist, you can now
-invite the Atomist Bot to your Slack team.  The Atomist Bot provides a
-chat-based interface to Atomist's capabilities.
+Atomist integrates with [Slack][slack] via the Atomist Bot, which
+provides a chat-based interface to Atomist's capabilities.
+
+[slack]: https://slack.com/ (Slack)
 
 After you filled out the [Atomist Alpha][alpha] form, you should have
 received an email invitation that looks something like the following:
```

Comment:
Image updates will be done once the atomist-docs GitHub/Slack teams are in place. There's already a lot of out-of-date and inconsistent imagery.